include/arch/arm/cortex_m/misc.h

```diff
@@ -24,6 +24,10 @@ extern void k_cpu_idle(void);
 extern u32_t z_timer_cycle_get_32(void);
 #define z_arch_k_cycle_get_32()	z_timer_cycle_get_32()
 
+#if CONFIG_ALARM
+extern u64_t z_timer_cycle_get_64(void);
```

Comment:
Could we introduce k_cycle_get_64() instead of this internal function?

Reply:
Yes.